src/main/java/uk/gov/hmcts/reform/wataskconfigurationapi/auth/role/RoleAssignmentService.java

```diff
@@ -63,13 +64,16 @@ public RoleAssignmentResource performSearch(QueryRequest queryRequest) {
         }
     }
 
-    private QueryRequest buildQueryRequest(String caseId) {
-        return QueryRequest.builder()
+    private MultipleQueryRequest buildQueryRequest(String caseId) {
+        QueryRequest queryRequest = QueryRequest.builder()
             .roleType(singletonList(RoleType.CASE))
             .roleName(singletonList("tribunal-caseworker"))
```

Comment:
Not part of this PR but are we hard coding the roleName?  What if we want to query for a different role(s).

Reply:
I believe this was part of a previous ticket to find if someone should be auto-assigned if we wanted to support more roles might need to review if it is a change of requirements.